src/System.Private.CoreLib/shared/System/Globalization/CultureData.Windows.cs

```diff
@@ -256,7 +256,7 @@ private int GetFirstDayOfWeek()
             Debug.Assert(!GlobalizationMode.Invariant);
             Debug.Assert(regionName != null);
 
-            EnumLocaleData context = new EnumLocaleData();
+            EnumLocaleData context = default;
```

Comment:
In cases like this one, it would look better to omit the `= default` completely - it is not necessary.

Or write it as: `var context = new EnumLocaleData() { cultureName = null, regionName = regionName };`

Reply:
I agree.  It's just really hard to see at the call site, we don't currently have an analyzer for it, etc.